templates/my.cnf.erb

```diff
@@ -0,0 +1,9 @@
+[client]
```

Comment:
I do not have anything against adding external dependencies, but please keep this improvement for the next release; we do have quite a lot of changes in this one.